.gitignore

```diff
@@ -44,3 +44,170 @@ site
 
 # Checks
 .tfsec
+
```

Comment:
Optional - this could be put under the Ray example directory in its own `.gitignore`, it will get merged with the one from the project root when evaluating what to ignore